Cargo.toml

```diff
@@ -84,12 +84,15 @@ usvg = "0.43.0"
 winresource = "0.1.17"
 xmlwriter = "0.1.0"
 # Enabling feature > v20_9 causes linker errors on mingw
-poppler-rs = { version = "0.23.0", features = ["v20_9"] }
+poppler-rs = { version = "0.24.1", features = ["v20_9"] }
 
 [patch.crates-io]
 # once a new piet (current v0.6.2) is released with updated cairo and kurbo deps, this can be removed.
-piet = { git = "https://github.com/linebender/piet", rev = "02eb5f0152e893626c43980bf37eeb0ccb1acb46" }
-piet-cairo = { git = "https://github.com/linebender/piet", rev = "02eb5f0152e893626c43980bf37eeb0ccb1acb46" }
+piet = { git = "https://github.com/flxzt/piet", rev = "a480bc20bd0d235928d46b009dae54e52e32767a" }
```

Comment:
kurbo's version seems to be updated in a later commit, https://github.com/flxzt/piet/commits/bump-cairo-pango/, you could specify `branch = "bump-cairo-pango"`

on a side note, could you add this commit https://github.com/linebender/piet/pull/572/commits/091ddfd1fa92c3454cff0294be3ce2d835d95556 to the branch :eyes:

Reply:
oops, copied the wrong commit. Fixed it now.

I don't want to directly add your changes to the existing version-bump branch.
Maybe we could wait for some feedback and if it is clear that it will be merged,
we can include it.
But if we would add it now, and implement the dash feature based on your feature branch, we would possibly have to reverse the changes before doing a release. Because I want to remove the patch before releasing a new version.